data/settings.js

```diff
@@ -4865,11 +4865,13 @@ query looking up the dict key.`
 	},
 
 	dict_map_value_name: {
-		tags: [ 'dict', 'dict-sql' ],
+		tags: [ 'dict', 'dict-sql', 'dict-ldap' ],
 		values: setting_types.STRING,
 		seealso: [ 'dict_map_value' ],
 		text: `
-Field in the SQL table to use for the [[setting,dict_map_value]].`
+Field in the SQL table to use for the [[setting,dict_map_value]].
+or
+Value to be returned from an ldap search, as a var-expression.`
```

Comment:
Maybe a list:
 * sql: Field in ...
 * ldap: Value to ...

Also use `[[link,settings_variables,variable-expression]]`